pkg/virt-launcher/notify-client/client.go

```diff
@@ -460,7 +461,8 @@ func (n *Notifier) StartDomainNotifier(
 	}()
 
 	domainEventLifecycleCallback := func(c *libvirt.Connect, d *libvirt.Domain, event *libvirt.DomainEventLifecycle) {
-		log.Log.Infof("DomainLifecycle event %d with reason %d received", event.Event, event.Detail)
+
+		log.Log.Infof("DomainLifecycle event %s received", event.String())
```

Comment:
Please, keep this. I found it useful in the past.

Reply:
i changed this to have the human readable string as well as the event and detail codes